inception/inception-search-mtas/src/main/java/de/tudarmstadt/ukp/inception/search/index/mtas/MtasDocumentIndex.java

```diff
@@ -909,6 +1130,15 @@ private void indexDocument(String aDocumentTitle, long aSourceDocumentId,
         doc.add(new StringField(FIELD_TITLE, aDocumentTitle, Field.Store.YES));
         doc.add(new StringField(FIELD_USER, aUser, Field.Store.YES));
         doc.add(new StringField(FIELD_TIMESTAMP, timestamp, Field.Store.YES));
+        // FieldType fieldTypeForContent = new FieldType();
```

Comment:
What is this?

Reply:
Forgot to delete code from my first approach.